go/store/nbs/archive_writer.go

```diff
@@ -330,6 +346,9 @@ func (aw *archiveWriter) writeFooter() error {
 	aw.bytesWritten += archiveFileSigSize
 	aw.workflowStage = stageFlush
 
+	aw.footerCheckSum = sha512Sum(aw.output.GetSum())
```

Comment:
This is...not a lot of entropy into the footerCheckSum? The only non-deterministic elements are the number of `stagedChunks` and the number of groups that those groups went into (determining byte span count?) Or maybe I'm thinking about it wrong? We should probably use a hash of the whole index or something...

Reply:
The footer contains the SHA-512 for the data, index, and metadata. Seems sufficient to me, but willing to discuss.

Reply:
Yeah, that's totally fine. I misread the `writeChuckSums` call up above since its body wasn't in the commit diff.